src/Stratis.Bitcoin.Features.PoA/PoAFeature.cs

```diff
@@ -70,12 +74,19 @@ public override Task InitializeAsync()
 
             this.federationManager.Initialize();
 
+            if (this.federationManager.IsFederationMember)
```

Comment:
Concerning the keys:
At the moment a federation member will have to have a file in a certain format, in the right place, and the node will start to mine by itself because discovery of the file means they're a miner. It feels counter-intuitive to me and isn't in keeping with how other miners do it.

A better approach is to 
- have the keys in the config file
- ask the miners to start the node with -mine.

It's simpler, clearer for the miners and easier to test.

Reply:
The reason behind having this as a separated file is mostly the ability to back it up easily. Also separated file feels more important than a config value. I want ppl to treat it like a wallet file 🙂

Reply:
Still, it's not a wallet file and these should be backed up somewhere else. A couple of values are easier to backup in a password manager than a file.

Reply:
Ok, reasonable. 
I've added those changes to my TODO list. Will do them when other tasks are completed.